src/style/reset.css

```diff
@@ -0,0 +1,11 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
the body doesn't have default padding. 

And it makes no sense to set `box-sizing: border-box;` to the body. If you want apply to all elements, set it under * selector